dummy-relying-party/frontend/src/main.ts

```diff
@@ -0,0 +1,186 @@
+import { AuthClient } from "@dfinity/auth-client";
```

Comment:
Is there some test for the front-end as well?
It would be great if we had e2e testing to be sure that we don't accidentally break the demo apps in the future.

Reply:
Ah, since there is no dummy issuer yet, you cannot do the e2e testing, right?
So this will be added later?

Reply:
That's correct. The demo issuer is already in another branch. This is the order I intend to follow:

1. Dummy relying party.
2. Dummy issuer (including canister tests with PocketIC)
3. E2E tests using both.
4. Expose wasms to releases page with Github actions.
5. Improve UI for dummy relying party.